triangle.js

```diff
@@ -0,0 +1,30 @@
+const pry = require('pryjs')
+
+const isPositive = function(int) {
+    return int > 0
+}
+
+const hasHypot = function(sides) {
+    let sorted = sides.sort(function(a, b) {
+        return a - b
+    })
+    return (sorted[2] ** 2) === (sorted[1] ** 2) + (sorted[0] ** 2)
+}
+
+const isIsoceles = function(sides) {
+    return (sides[0] === sides[1] && sides[1] === sides[2])
+}
+
+
+const isTriangle = function(sideA, sideB, sideC) {
```

Comment:
Nice! Have you looked into arrow functions at all? One line returns are great for it, especially since you're using `const`.